src/parsec_security/rust_client_threat_model/threat_model.md

```diff
@@ -130,7 +130,7 @@ sensitive information.
 | ID | Justification                                                                                                                                                                            | Details                                                                                                                                                                    |
 |----|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | 0  | Parsec interface uses an IPC mechanism respecting confidentiality and integrity of messages transmitted between the clients and the service (once the initial connection has been made). | Unix Domain Socket: the sockets used on the client and service side for the communication are represented by file descriptors that are only accessible by those processes. |
-| 1  | The Parsec client is coded with safety in mind and is tested extensively.                                                                                                                | [Open](https://github.com/parallaxsecond/parsec-book/issues/35)                                                                                                            |
+| 1  | The Parsec client is coded with safety in mind and is tested extensively.                                                                                                                | [Done](https://github.com/parallaxsecond/parsec-book/issues/35)                                                                                                            |
```

Comment:
This one is a bit dubious.. I did not really add details over what kind of tests are done in the Rust clients itself..